src/processor.js

```diff
@@ -106,6 +107,7 @@ function createProcessor (node, opts) {
           }
         }),
         pull.map(function (x) { return String(x) }),
+        pull.map(x => zlib.gzipSync(x)),
```

Comment:
Please use a regular anonymous function (e.g. 'function (x) { return x }') rather than an arrow function. That way, pando will work on older versions of Node without additional tooling such as babel.